test/passing/funsig.ml

```diff
@@ -26,16 +26,16 @@ val f :
 type t =
   | Cstr of
       (   xxxxxxxxxxxxxxxxxxxxxxxxx
```

Comment:
Is this a regression ?
Otherwise, there is an extra space before the first type.

Reply:
Fixed